geomesa-gs-monitor-elasticsearch/src/main/scala/org/geomesa/gs/monitor/elastic/ElasticRequestDataListener.scala

```diff
@@ -0,0 +1,140 @@
+/***********************************************************************
+ * Copyright (c) 2013-2020 Commonwealth Computer Research, Inc.
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the GNU GENERAL PUBLIC LICENSE,
+ * Version 2 which accompanies this distribution and is available at
+ * https://opensource.org/licenses/GPL-2.0.
+ ***********************************************************************/
+
+package org.geomesa.gs.monitor.elastic
+
+import java.lang.reflect.Type
+import java.net.{MalformedURLException, URL}
+import java.util.Date
+
+import com.google.gson._
+import com.typesafe.scalalogging.LazyLogging
+import org.apache.http.HttpHost
+import org.elasticsearch.action.ActionListener
+import org.elasticsearch.client.{RequestOptions, RestClient, RestHighLevelClient}
+import org.elasticsearch.action.index.{IndexRequest, IndexResponse}
+import org.elasticsearch.common.xcontent.XContentType
+import org.elasticsearch.action.DocWriteResponse
+import org.geoserver.monitor.{RequestData, RequestDataListener}
+import org.opengis.geometry.BoundingBox
+
+import scala.collection.mutable.ArrayBuffer
+
+
+
+class ElasticRequestDataListener extends RequestDataListener with LazyLogging {
+  import ElasticRequestDataListener.gson
+  val envVars = sys.env.getOrElse("ELASTICSEARCH_HOST", null).split(',')
```

Comment:
this would throw an NPE if the env var isn't set... could handle it a little cleaner and throw a clearer exception (or just pass it down to the error handling checks below)